components/raftstore/src/store/fsm/apply.rs

```diff
@@ -4410,6 +4410,11 @@ where
     }
 
     fn handle_tasks(&mut self, apply_ctx: &mut ApplyContext<EK>, msgs: &mut Vec<Msg<EK>>) {
+        fail_point!(
+            "only_apply_for_region_1_on_store_1",
```

Comment:
Please use a more generic fail point name here.

Reply:
But this fail-point itself is not generic as it has a condition. I saw many other fail-points naming like this